cie_infra/controllers/controllers.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
The header `# -*- coding: utf-8 -*-` is not mandatory anymore in python3, as by default in python 3 file are uft8 by default. So it's better to remove it.